flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSinkExtended.java

```diff
@@ -57,6 +57,9 @@ public class TestFlinkIcebergSinkExtended extends TestFlinkIcebergSinkBase {
   private final boolean partitioned = true;
   private final int parallelism = 2;
   private final FileFormat format = FileFormat.PARQUET;
+  // FIXME (before merge): should it be parameterized test or hardcoded value with some comment that
```

Comment:
@pvary WDYT? Should I extract parameter tests or keep this hardcoded as other things in this test?

Reply:
Let's agree on the general approach first, before going into these details.

Reply:
Good plan.